utils/hct/hctdb.py

```diff
@@ -8381,6 +8381,7 @@ def process_arg(desc, idx, done_args, intrinsic_name):
             component_list = "LICOMPTYPE_ANY"
             rows = "1"
             cols = "1"
+            only = ""
```

Comment:
I understand why you would choose only for a variable name, but I'm wondering if there could be a better way to name this.